src/main/java/org/folio/service/inventory/InventoryManager.java

```diff
@@ -691,8 +691,7 @@ private Error buildErrorWithParameter(String value, ErrorCodes errorCode) {
   }
 
   public String buildProductIdQuery(ProductId productId) {
-    return String.format(
-      "(identifiers adj \"\\\"identifierTypeId\\\": \\\"%s\\\"\" " + "and identifiers adj \"\\\"value\\\": \\\"%s\\\"\")",
+    return String.format("(identifiers =/@identifierTypeId=\"%s\"/@value \"%s\")",
```

Comment:
Based on requirements we should remove from search identifiers with invalid ISBN or ISSN type.
Example : do not include the following identifiers- with types "Invalid ISBN" and "Invalid ISSN" in the search of the existing instance.
Do we meet origin requirements or requirements  were changed?

Reply:
Requirements were changed (it is no longer visible in the description, but at some point ignoring the invalid ISBN type was crossed out). The issue was not that invalid ISBNs were taken into account, it was a mismatch when both valid and invalid ISBNs were used. Ignoring invalid ISBNs would have been a workaround for some cases, but fixing the instance matching is a better fix.

Reply:
Thanks. Good catch